tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -545,7 +545,8 @@ static struct device_info boards[] = {
 			"SupportList:\r\n"
 			"WBS510(TP-LINK|UN|N300-5):1.20\r\n"
 			"WBS510(TP-LINK|US|N300-5):1.20\r\n"
-			"WBS510(TP-LINK|EU|N300-5):1.20\r\n",
+			"WBS510(TP-LINK|EU|N300-5):1.20\r\n"
+			"WBS510(TP-LINK|CA|N300-5):1.20\r\n",
```

Comment:
So, how sure are you about the "1.20" here?

Reply:
Yes in this case the 1.20 is correct.
I just added the Canadian Model